docs/recipes/react.md

```diff
@@ -0,0 +1,69 @@
+# React testing
+
+With 0.10.0, AVA stopped including your `.babelrc` (for reasons outlined in [this PR](https://github.com/sindresorhus/ava/pull/398)), which means you can't use JSX in your test/spec files. Note that this doesn't apply to your source files, they can still be written in JSX!
```

Comment:
In the interest of future-proofing this documentation, a reference to the relevant issue might be helpful here. Can't seem to find it right now though.